### PR TITLE
Update CHANGELOG for scratch option

### PR DESCRIPTION
**Pull request by loosebazooka**
forgot to update changelog... again...

### CODE REVIEW

File: jib-maven-plugin/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ All notable changes to this project will be documented in this file.
 ### Added
 
 - Can now containerize a JAR artifact instead of putting individual `.class` and resource files with `<containerizingMode>packaged` ([#1746](https://github.com/GoogleContainerTools/jib/pull/1746/files))
+- Can now use `<image><from>scratch</from></image>"` to use the scratch (empty) base image for builds. ([#1794](https://github.com/GoogleContainerTools/jib/pull/1794/files))
```
Comment (chanseokoh) [inline jib-maven-plugin/CHANGELOG.md]: We've been skipping closing tags for brevity. (Remove the stray double-quote attached to it too.)

File: jib-gradle-plugin/CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,7 @@ All notable changes to this project will be documented in this file.
 ### Added
 
 - Can now containerize a JAR artifact instead of putting individual `.class` and resource files with `jib.containerizingMode = 'packaged'` ([#1760](https://github.com/GoogleContainerTools/jib/pull/1760/files))
+- Can now use `jib.image.from = "scratch"` to use the scratch (empty) base image for builds. ([#1794](https://github.com/GoogleContainerTools/jib/pull/1794/files))
```
Comment (briandealwis) [inline jib-gradle-plugin/CHANGELOG.md]: It's `jib.from.image`

File: jib-maven-plugin/CHANGELOG.md (hunk 1 above)
Comment (briandealwis) [inline jib-maven-plugin/CHANGELOG.md]: `<from><image>`

File: jib-gradle-plugin/CHANGELOG.md (hunk 2 above)
Comment (chanseokoh) [inline jib-gradle-plugin/CHANGELOG.md]: We've been using single quotes for a config value.

**Pull request merged**